builds/posix/Makefile.in.examples

```diff
@@ -67,8 +67,16 @@ export PATH
 #endif
 GPRE_FLAGS=	-m -z -n
 
-LIB_LINK_RPATH = $(call LIB_PLATFORM_RPATH,$(if $(subst intl,,$(1)),$(LIB),$(LIB)/../intl))
-LINK_DARWIN_RPATH = -Wl,-rpath,@loader_path/../$(TARGET)/firebird
+ifeq (@USE_RPATH@,1)
```

Comment:
Redefine LIB_PLATFORM_RPATH in minugw.prefix instead.

Reply:
ok